### PR TITLE
Reomve, closes #1385

### LINKED ISSUE #1385: Could you please consider adding "reomve" as a version of "remove"?
Looks like the following correction would be useful:

reomve -> remove

and it looks like "reomve" is not an existing word that would have any meaning.

### PR DESCRIPTION
**Pull request by peternewman**
Supersedes https://github.com/luzpaz/codespell/pull/1

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -919,8 +919,13 @@ akkumulation->accumulation
 akkumulative->accumulative
 akkumulator->accumulator
 aks->ask
+aksed->asked
+akss->asks
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Suggest ass

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -3179,6 +3184,10 @@ blaim->blame
 blaimed->blamed
 blak->black, blank,
 blanace->balance
+blance->balance, glance,
+blanced->balanced, glanced,
+blances->balances, glances,
+blancing->balancing, glancing,
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Suggest lanc*

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -21118,13 +21161,14 @@ similiarity->similarity
 similiarly->similarly
 similiarty->similarity
 similiary->similarity
+simily->simile, smiley, similarly, simply,
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: I think `similarly, simply` should swap order

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -21118,13 +21161,14 @@ similiarity->similarity
...
 simlar->similar
 simlarlity->similarity
 simlarly->similarly
 simliar->similar
 simlicity->simplicity
 simlified->simplified
-simly->simply
+simly->simply, smiley, simile,
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: I think `smiley, simile` should swap order

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: bce81df

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: bce81df

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: bce81df

**Pull request merged**